nitro-protocol/package.json

```diff
@@ -1,12 +1,19 @@
 {
   "name": "@statechannels/nitro-protocol",
   "version": "2.0.0-alpha.0",
-  "description": "Smart contracts and typescript libraries for nitro v2",
-  "main": "index.js",
+  "author": "statechannels.org",
+  "bugs": "https://github.com/statechannels/go-nitro/issues",
+  "description": "Smart contracts and typescript libraries for nitro state channel protocol.",
+  "main": "lib/src/index.js",
+  "homepage": "https://github.com/statechannels/go-nitro",
```

Comment:
Is `docs.statechannels.org` or just `statechannels.org` suitable here? Repo is also included in the metadata.

Reply:
Good point. I think the docs website maybe a good shout.